torch_xla/csrc/reduction.cpp

```diff
@@ -137,8 +137,11 @@ xla::XlaOp BuildBinaryCrossEntropy(xla::XlaOp input, xla::XlaOp target,
         XlaHelpers::ScalarBroadcast<float>(1.0, input_shape, target.builder());
   }
   xla::XlaOp one = xla::One(input.builder(), input_shape.element_type());
-  xla::XlaOp result = -xweight * (target * xla::Log(input) +
-                                  (one - target) * xla::Log(one - input));
+  xla::XlaOp log_bound = XlaHelpers::ScalarValue(
+      -100, input_shape.element_type(), input.builder());
```

Comment:
```c++
static const float kLogBound = -100;
```

Reply:
out of curiosity, when do we prefix a variable with `k`? I have seen this many times in our code base but didn't know why.